app/helpers/window.js

```diff
@@ -15,17 +15,43 @@ const autoResize = (dynamicWindow) => {
     }
   }
 
-  dynamicWindow.webContents.on('did-finish-load', () => {
-    const updateHeight = () => {
-      if (!dynamicWindow) { return }
-      dynamicWindow.webContents.executeJavaScript('document.body.children[0].offsetHeight', (mainContentHeight) => {
-        resize(mainContentHeight)
-      })
+  const updateHeight = () => {
+    if (!dynamicWindow) { return }
+    dynamicWindow.webContents.executeJavaScript('document.body.children[0].offsetHeight', (mainContentHeight) => {
+      resize(mainContentHeight)
+    })
+  }
+
+  let updateHeightIntervalId = null
+  const clearUpdateHeightInterval = () => {
+    if (updateHeightIntervalId) {
+      clearInterval(updateHeightIntervalId)
+      updateHeightIntervalId = null
+    }
+  }
+
+  // register updateHeight interval only when dynamicWindow is visible
+  const registerUpdateHeightInterval = () => {
+    clearUpdateHeightInterval()
+    if (dynamicWindow.isVisible()) {
+      // avoid 125ms delay and redraw the window right away
+      updateHeight()
+      updateHeightIntervalId = setInterval(updateHeight, 125)
     }
-    const id = setInterval(updateHeight, 125)
-    dynamicWindow.on('closed', () => {
-      clearInterval(id)
+  }
+
+  dynamicWindow.webContents.on('did-finish-load', () => {
+    // remove the interval as soon as the dynamicWindow is not visible or destroyed
+    dynamicWindow.on('closed', clearUpdateHeightInterval)
+    dynamicWindow.on('hide', clearUpdateHeightInterval)
+
+    // reregister the interval as soon as the window is visible
+    dynamicWindow.on('show', () => {
+      registerUpdateHeightInterval()
     })
+
+    // if the window is already visible, the interval should be set
+    registerUpdateHeightInterval()
```

Comment:
I already do this check in the function, to be sure that we dont create the interval when the window is hidden, in this case we will not receive a hide interval because it is already hidden